### ISSUE TITLE
input type=color not properly announced with JAWS + Chromium

### ISSUE THREAD

**Issue by scottaohara** [FreedomScientific/standards-support#227]
### Summary
JAWS 2019 (june) paired with Chrome (75) do not consistently announce a color input's accessible name, if anything at all.

1. With JAWS enabled, go to https://codepen.io/scottohara/full/dxvPbj
2. Attempt to find the color input via <kbd>ins</kbd> + <kbd>F5</kbd>
3. Attempt to find the color input by use of <kbd>tab</kbd> key navigation
4. Attempt to find the color input by use of virtual cursor navigation.

### Expected result
Would expect to find a form field with the accessible name "choose"

### Actual result
re: 2. Input not listed in form field dialog.
re: 3. Keyboard focus will navigate to the color input, but JAWS will not announce it's accessible name. Instead JAWS repeats the previously focused element's accName when focusing the color input.
re: 4. Virtual Cursor will find the "choose" text. Hitting <kbd>Space</kbd> or <kbd>Enter</kbd> **WILL** open the browser's color picker dialog.

### Example
https://codepen.io/scottohara/full/dxvPbj

### Additional Information
Chromium exposes the element with the following roles: 
MSAA editable text.   
IA2: Color Chooser.   
UIA: UIA_Edit  (edge)

#### JAWS version and build number
JAWS 2019.1906.10

 
#### Operating System and version
Windows 10

#### Browser and version:
Chrome 75
Edge 77 (dev)

For comparison, JAWS with Firefox (68.0.1) will announce the color input as a "button".

**Comment by JAWS-test** [comment]
The problem also occurs in my test environment as described above.

**Comment by lamador** [comment]
The issue was reproducible using the latest version of JAWS (2020.2006.12), Chrome (83.0.4103.116) and Edge (83.0.478.45).

**Comment by JAWS-test** [comment]
The problem continues to occur (JAWS 2021). With NVDA, the problem does not occur despite the wrong role ColorWell

**Comment by JAWS-test** [comment]
With JAWS 2023, the problems no longer occur. The button is output correctly with its label.

It is not perceptible that the button is for colour selection and which is the currently selected colour, but these are only minor problems.

**Comment by stevefaulkner** [comment]
thanks @JAWS-test
